pyproject.toml

```diff
@@ -39,7 +39,7 @@ sphinx = ">=5.0"
 colorama = ">=0.4.6"
 docutils = ">=0.15"
 pathlib = ">=1.0"
-weasyprint = ">=60.0, <61.0"
+weasyprint = ">=62.0"
```

Comment:
Please change the Python version to 3.10 and above since `weasyprint >=62.0` requires `Python >=3.9`
``` toml
python = ">=3.10.0,<4.0"
```
After the change, you can update the `poetry.lock` using `poetry update`.

Reply:
THX! I updated as you suggested.